.github/workflows/osdk_publish.yml

```diff
@@ -21,3 +21,34 @@ jobs:
           args: --no-verify
           registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
           ignore-unpublished-changes: true
+
+  osdk-publish-dockerhub:
```

Comment:
I suggest doing it in the `docker_build.yml` since:
 - you can reuse the fetch versions code there;
 - seems more consistent since the docker image is only used for testing and this image is not designed to be used by the OSDK user.

Reply:
Moved to `docker_build.yml`.